lib/streamlit/web/server/browser_websocket_handler.py

```diff
@@ -102,7 +102,25 @@ def open(self, *args, **kwargs) -> Optional[Awaitable[None]]:
         else:
             user_info["email"] = email
 
-        self._session_id = self._runtime.create_session(self, user_info)
+        existing_session_id = None
+        try:
+            ws_protocols = [
+                p.strip()
+                for p in self.request.headers["Sec-Websocket-Protocol"].split(",")
+            ]
+
+            if len(ws_protocols) > 1:
+                existing_session_id = ws_protocols[1]
```

Comment:
Can you add a comment above the code explaining why we look for the existing_session_id in the "Sec-Websocket-Protocol" headers?

Reply:
👍 -- I've already written an explanation for why this is necessary in the docstring of `select_subprotocol` in this file, so I'll just add a comment giving a pointer to that.